docs/_config.yml

```diff
@@ -5,14 +5,14 @@ markdown: kramdown
 kramdown:
   entity_output: as_input
 
-latest_leaflet_version: 1.9.4
+latest_leaflet_version: 1.9.2
```

Comment:
Can you explain why you downgraded the version?
This looks like a mistake to me but I could be wrong...

Reply:
I am really sorry for this, I didn't notice it.

Reply:
> I am really sorry for this, I didn't notice it.

You don't have to be sorry. Everyone makes mistakes.